src/main.rs

```diff
@@ -10,6 +10,7 @@ fn main() {
         dotenv::var("DISCORD_TOKEN").expect("DISCORD_TOKEN must be set"),
         dotenv::var("AUTHORIZED_USERS").expect("AUTHORIZED_USERS must be set"),
         dotenv::var("HEROKU_API_KEY").expect("HEROKU_API_KEY must be set"),
+        dotenv::var("NO_AUTH_COMMANDS").expect("NO_AUTH_COMMANDS must be set"),
```

Comment:
I don't think using an environment variable to list the commands that don't require authentication is the approach we should go with. The list of commands we want to be unauthenticate should not be different between environments, and any change there should go through code review. A constant somewhere in the code would be better in my opinion.